apps/coscrad-frontend/src/components/resources/videos/video-index.presenter.tsx

```diff
@@ -0,0 +1,44 @@
+import { IVideoViewModel, MultilingualTextItemRole } from '@coscrad/api-interfaces';
+import { VideoIndexState } from '../../../store/slices/resources/video';
+import { HeadingLabel, IndexTable } from '../../../utils/generic-components/presenters/tables';
+import { CellRenderersDefinition } from '../../../utils/generic-components/presenters/tables/generic-index-table-presenter/types/cell-renderers-definition';
+import { renderAggregateIdCell } from '../utils/render-aggregate-id-cell';
+import { renderMediaLengthInSeconds } from '../utils/render-media-length-in-seconds-cell';
+
+export const VideoIndexPresenter = ({ entities: videos }: VideoIndexState): JSX.Element => {
+    const headingLabels: HeadingLabel<IVideoViewModel>[] = [
+        { propertyKey: 'id', headingLabel: 'Link' },
+        {
+            propertyKey: 'lengthMilliseconds',
+            headingLabel: 'Video Length',
+        },
+        { propertyKey: 'name', headingLabel: 'Name' },
+    ];
+
+    const cellRenderersDefinition: CellRenderersDefinition<IVideoViewModel> = {
+        id: renderAggregateIdCell,
+        lengthMilliseconds: ({ lengthMilliseconds }: IVideoViewModel) =>
+            renderMediaLengthInSeconds(lengthMilliseconds),
+        name: ({ name }: IVideoViewModel) => {
+            const originalItem = name.items.find(
```

Comment:
We might want to have a `MultilingualNameCellRenderer` so this is consistent across resources of different types.

Reply:
done